packages/dd-trace/src/appsec/sdk/track_event.js

```diff
@@ -58,6 +60,10 @@ function trackEvent (eventName, fields, sdkMethodName, rootSpan, mode) {
     [MANUAL_KEEP]: 'true'
   }
 
+  if (isStandaloneEnabled()) {
+    tags[APPSEC_PROPAGATION_KEY] = 1
```

Comment:
doesn't setting tags as numbers make them go into metrics instead of tags by default ? should it be `'1'` instead ?